lib/main.js

```diff
@@ -96,6 +97,7 @@ globals.setImmediate(() => {
 });
 
 module.exports = runner.chain;
+module.exports.CallSite = CallSite;
```

Comment:
That would require putting it as it's own file in the directory root though, next to `index` and `cli`, wouldn't it?